app/static/middle.css

```diff
@@ -0,0 +1,288 @@
+@import url('https://fonts.googleapis.com/css2?family=Dosis&family=Poppins:wght@300&family=Roboto:wght@300&display=swap');
+
+body {
+  font-family: 'Poppins', sans-serif;
+}
+
+.section-elements {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.middle-section {
+  /* width: 295px;
+  height: 36px; */
```

Comment:
- [ ] Kindly remove any commented out and unused/ unwanted pieces of code from your code base. It makes it look cluttered and messy.